### PR TITLE
fix: data column in SQL lab left panel open by default 

### PR DESCRIPTION
**Pull request by AAfghahi**
### SUMMARY
Fix to Left Panel tables being auto-expanded when added. This was a feature they had previous to the switch to antd Collapse. 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

https://user-images.githubusercontent.com/48933336/111169955-d2da7000-8579-11eb-856f-0c3b391d2430.mov

### ADDITIONAL INFORMATION

- [ ] Has associated issue: https://github.com/apache/superset/issues/13366

### CODE REVIEW

File: superset-frontend/src/SqlLab/components/SqlEditorLeftBar.jsx [hunk 1]
```diff
@@ -147,8 +147,9 @@ export default class SqlEditorLeftBar extends React.PureComponent {
         <StyledScrollbarContainer>
           <StyledScrollbarContent contentHeight={tableMetaDataHeight}>
             <Collapse
-              ghost
-              expandIconPosition="right"
+              activeKey={this.props.tables
+                .filter(({ expanded }) => expanded)
+                .map(({ id }) => id)}
```
Comment (betodealmeida) [inline superset-frontend/src/SqlLab/components/SqlEditorLeftBar.jsx:152]: I love Javascript's object unpacking, it's really elegant.

**Review by betodealmeida — APPROVED** [review]

**Comment by junlincc** [comment]
@michael-s-molina  ^ fyi collapse default behavior is not all the same in the product. i should have caught it. my bad. 🙏

**Comment by michael-s-molina** [comment]
> @michael-s-molina ^ fyi collapse default behavior is not all the same in the product. i should have caught it. my bad. 🙏

No problem at all 😉. 

Thank you all for the fix.

**Comment by AAfghahi** [comment]
@eschutho I was thinking about what the most effective way of testing that expanded is true when the query is added. I decided that checking the reducer would be the most effective way of making sure that when the action is taken, that the new table has an expanded value of true. Is this the correct thinking? Or should I be testing sqlLab action as well? 

In the latter case, looking through https://redux.js.org/recipes/writing-tests says that I should be importing the reducer and checking it (that's why I decided to just test the reducer itself). Am I thinking of this correctly?

Lastly --long note sorry-- I was thinking about testing the functionality of adding a column. Currently, I wrote an RTL test that checks to make sure that a table with expanded = true is shown in the document. I wanted to add a test that tests the functionality of clicking the element (though there is a spec written for the Collapse panel component that checks this, so maybe not necessary).

Also, I wanted to write a test that has you adding a table and that should come in expanded. The problem that I ran into here was that I think that I'd have to write a dataset/schema/table combination and then have fetchMocks that adds them to initial state? Does that sound correct? Would the test reside in SqlLeftEditor bar? Is this test necessary?

ahh sorry for the note, been thinking about this a lot.

**Review by hughhhh — APPROVED** [review]

**Pull request merged**